Remove optional test and dev dependencies we do not have packaged.

--- pyfakefs-6.2.0/pyproject.toml.orig
+++ pyfakefs-6.2.0/pyproject.toml
@@ -56,12 +56,8 @@
 [dependency-groups]
 dev = [
     "pytest>=6.2.5",
-    "pre-commit>=4.1.4",
 ]
 extra = [
-    "pandas==2.3.3",
-    "xlrd==2.0.2",
-    "openpyxl==3.1.5",
 ]
 
 [project.optional-dependencies]
